src/util/system.cpp

```diff
@@ -820,7 +841,7 @@ static std::string TrimString(const std::string& str, const std::string& pattern
     return str.substr(front, end - front + 1);
 }
 
-static bool GetConfigOptions(std::istream& stream, std::string& error, std::vector<std::pair<std::string, std::string>> &options)
+static bool GetConfigOptions(std::istream& stream, std::string& error, std::vector<std::pair<std::string, std::string>> &options, std::set<std::string> &sections)
```

Comment:
nit, space should be after `&` for new code.

Reply:
Oh, thanks.

Reply:
done.